docs/index.rst

```diff
@@ -48,18 +48,20 @@ and simulation to real-world production deployment.  Some of the key components
  - **Management tools** for secure provisioning and deployment, orchestration, and management
  - **Specification-based API** for extensibility
 
-Learn more in the :ref:`FLARE Overview <flare_overview>`, :ref:`What's New <whats_new>`, and the
-:ref:`User Guide <user_guide>` and :ref:`Programming Guide <programming_guide>`.
+Learn more about FLARE features in the :ref:`FLARE Overview <flare_overview>` and :ref:`What's New <whats_new>`.
 
 Getting Started
 ===============
 For first-time users and FL researchers, FLARE provides the :ref:`FL Simulator <fl_simulator>` that allows you to build, test, and deploy applications locally.
 The :ref:`Getting Started <getting_started>` guide covers installation and walks through an example application using the FL Simulator.
+Additional examples can be found at the :ref:`Examples Applications <example_applications_algorithms>`, which showcase different federated learning workflows and algorithms on various machine learning and deep learning tasks.
 
-When you are ready to for a secure, distributed deployment, the :ref:`Real World Federated Learning <real_world_fl>` section covers the tools and process
+FLARE for Users
+===============
+When you are ready to for a secure, distributed deployment, the :ref:`User Guide <user_guide>` and :ref:`Real World Federated Learning <real_world_fl>` sections covers the tools and process
 required to deploy and operate a secure, real-world FLARE project.
```

Comment:
User guide is for operating/interacting with FLARE, does not necessary mean secure and deployment?

So, suggest something like:

```
If you want to learn how to interacts with FLARE system, please refer to the :ref:`User Guide <user_guide>`.
When you are ready to for a secure, distributed deployment, the :ref:`Real World Federated Learning <real_world_fl>` sections covers the tools and process
required to deploy and operate a secure, real-world FLARE project.
```